<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rMainWindow</class>
 <widget class="QMainWindow" name="rMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>713</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>937</width>
    <height>713</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>rMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>851</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string>请选择要合并的数据库文件</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>85</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>文件路径：</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QListWidget" name="FileLists">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>811</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>851</width>
      <height>74</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string>请选择要合并的方式</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="rBtnSize">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>19</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>大小(fbk)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rBtnTime">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>19</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>时间(nbk)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>851</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>630</y>
      <width>851</width>
      <height>56</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pBtnStart">
       <property name="minimumSize">
        <size>
         <width>93</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBtnExit">
       <property name="minimumSize">
        <size>
         <width>93</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>自动</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>851</width>
      <height>78</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string>请选择要恢复文件所在的文件夹</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>文件夹：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="directoryComboBox">
       <property name="minimumSize">
        <size>
         <width>681</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="selectDirBtn">
       <property name="minimumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>485</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>341</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>请设置自动恢复数据库文件多少天一次：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="autoDaysEdit">
       <property name="maximumSize">
        <size>
         <width>113</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
